### PR TITLE
new icon

### PR DESCRIPTION
**Pull request by reesericci**
PR to add icon.

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -1511,6 +1511,13 @@ export const glyphs = {
...
     />
   ),
+  amogus: (
```
Comment (sampoder) [inline src/index.tsx:1514]:
```suggestion
  sus: (
```

**Review by sampoder — APPROVED** [review]
Ok Reese.

**Comment by reesericci** [comment]
we did it joe

**Comment by reesericci** [comment]
its not on the site tho

**Pull request merged**